<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>logindlg</class>
 <widget class="QDialog" name="logindlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>671</width>
    <height>583</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowFilePath">
   <string/>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>100</y>
     <width>414</width>
     <height>103</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0" colspan="2">
     <widget class="QLabel" name="label_3">
      <property name="styleSheet">
       <string notr="true">color: rgb(253, 211, 255);</string>
      </property>
      <property name="text">
       <string>流动地图：</string>
      </property>
     </widget>
    </item>
    <item row="1" column="4">
     <widget class="QComboBox" name="comboBox_4">
      <item>
       <property name="text">
        <string>开启</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>关闭</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QComboBox" name="comboBox_3">
      <item>
       <property name="text">
        <string>烈焰</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QLabel" name="label_5">
      <property name="styleSheet">
       <string notr="true">color: rgb(253, 211, 255);</string>
      </property>
      <property name="text">
       <string>用户名：</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QLabel" name="label_4">
      <property name="styleSheet">
       <string notr="true">color: rgb(253, 211, 255);</string>
      </property>
      <property name="text">
       <string>上帝模式：</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="styleSheet">
       <string notr="true">color: rgb(253, 211, 255);</string>
      </property>
      <property name="text">
       <string>难度  ：</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2" colspan="3">
     <widget class="QLineEdit" name="lineEdit"/>
    </item>
    <item row="0" column="2">
     <widget class="QComboBox" name="comboBox">
      <item>
       <property name="text">
        <string>简单</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>一般</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>困难</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QComboBox" name="comboBox_2">
      <item>
       <property name="text">
        <string>开启</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>关闭</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QLabel" name="label_2">
      <property name="styleSheet">
       <string notr="true">color: rgb(253, 211, 255);</string>
      </property>
      <property name="text">
       <string>地图:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="Start">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>310</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="rank">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>210</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(253, 211, 255);
text-decoration: underline;
font: italic 11pt &quot;Bahnschrift&quot;;</string>
   </property>
   <property name="text">
    <string>排行榜</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Help">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>210</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(253, 211, 255);
text-decoration: underline;
font: italic 9pt &quot;Bahnschrift&quot;;</string>
   </property>
   <property name="text">
    <string>说明</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Ours">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>210</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(253, 211, 255);
text-decoration: underline;
font: italic 9pt &quot;Bahnschrift&quot;;</string>
   </property>
   <property name="text">
    <string>关于作者</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Quit">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>20</y>
     <width>81</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
